docs/pages/database-access/guides/mysql-self-hosted.mdx

```diff
@@ -0,0 +1,257 @@
+
+Teleport uses mutual TLS for authentication to MySQL instances. As such,
+self-hosted MySQL instances must be configured with Teleport's certificate
+authority and a certificate/key pair that Teleport can validate.
+
+To create these secrets, use `tctl auth sign` command. Note that it requires a
+running Teleport cluster and [should be run](../../architecture/overview.mdx#tctl)
+on the auth server.
+
+```sh
+# Export Teleport's certificate authority and generate certificate/key pair
+# for host db.example.com with a one year validity period.
+$ tctl auth sign --format=db --host=db.example.com --out=server --ttl=8760h
+```
+
+Flag descriptions:
+
+* `--format=db`: instructs the command to produce secrets in the format suitable
+  for configuring a database server.
+* `--host=db.example.com`: server name to encode in the certificate, should
+  match the hostname Teleport will be connecting to the database at.
+* `--out=server`: name prefix for output files.
+* `--ttl=8760h`: certificate validity period.
+
+The command will create 3 files: `server.cas` with Teleport's certificate
+authority and `server.crt`/`server.key` with generated certificate/key pair.
+
+<Admonition type="note" title="Certificate Rotation">
+Teleport signs database certificates with the host authority. As such,
+when performing [host certificates rotation](../../admin-guide.mdx#certificate-rotation),
+the database certificates must be updated as well.
+</Admonition>
+
+## Configure MySQL Server
+
+To configure MySQL server to accept TLS connections, add the following to
+MySQL configuration file `mysql.cnf`:
+
+```conf
+[mysqld]
+require_secure_transport=ON
+ssl-ca=/path/to/server.cas
+ssl-cert=/path/to/server.crt
+ssl-key=/path/to/server.key
+```
+
+Additionally, MySQL database user accounts must be configured to require a
+valid client certificate:
+
+```sql
+CREATE USER 'alice'@'%' REQUIRE X509;
+ALTER USER 'alice'@'%' REQUIRE X509;
+```
+
+See [Configuring MySQL to Use Encrypted Connections](https://dev.mysql.com/doc/refman/8.0/en/using-encrypted-connections.html)
+in MySQL documentation for more details.
+
+## Configure Teleport
+
+Teleport Database Access is available starting from `6.0.0-alpha.1` pre-release.
+
+Download the appropriate version of Teleport for your platform from the table
+below, or visit our [downloads page](https://goteleport.com/teleport/download).
+
+{!docs/pages/preview/releases-table.mdx!}
+
+<Admonition type="warning" title="Version Warning">
+Note, pre-releases are not suitable for production usage!
+</Admonition>
+
+Follow the installation [instructions](../../installation.mdx).
+
+### Start Auth/Proxy Service
+
+Create a configuration file for a Teleport service that will be running
+auth and proxy servers:
+
+```yaml
+teleport:
+  data_dir: /var/lib/teleport
+  nodename: test
+auth_service:
+  enabled: "yes"
+  cluster_name: "test"
+  listen_addr: 0.0.0.0:3025
+  tokens:
+  - proxy,node,db:cbdeeab9-6f88-436d-a673-44d14bd86bb7
```

Comment:
let's never ever hardcode any tokens in our docs, you will find those everywhere in the internet

instead promote usage the most secure tokens - dynamic tokens with short ttl.

```
tctl nodes add --roles=db > /tmp/file
```

and then join
```
tctl start --token=/tmp/file
```